.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 python:
   - 2.7
-  - 3.3
   - 3.4
   - 3.5
   - 3.6
```

Comment:
Not strictly related, but pypy is in tox.ini, so it could be tested on Travis here, too.